public/js/createListing.js

```diff
@@ -5,7 +5,7 @@ $(function() {
     event.preventDefault();
 
     var title = $('#title-line').val().trim();
-    var address = $('#address-line').val().trim();
+    var address = $('#address-line-1').val().trim(); // update with all address form...
```

Comment:
Add the following to the form:
* Address One
* Address Two
* City
* State
* Zipcode